packages/crypto-wallet-core/src/validation/eth/index.ts

```diff
@@ -3,6 +3,25 @@ const utils = require('web3-utils');
 
 export class EthValidation implements IValidation {
   validateAddress(_network: string, address: string): boolean {
+    address = this.sanitizeEthereumUri(address);
```

Comment:
I'm somewhat worried about this change. We rely on this method to tell us whether an address is valid, so we can pass it into methods that rely on an address being provided. 

If we do this, we lose that and instead it's more of a "contains" a valid address